pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md

```diff
@@ -276,6 +331,28 @@ enterprise: false
 -   traefik-forward-auth: 1.0.4-4
 -   velero: 1.0.1-4
 
+### Version v1.4.4 - Released 28 May 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+|**Minimum** | 1.15.4 |
+|**Maximum** | 1.16.x |
+|**Default** | 1.16.9 |
+
+#### Improvements
+
+-   Change the default Azure region for new clusters to `eastus2` to improve stability.
+-   New `cluster.yaml` option `spec.kubernetes.imageRepository` to allow for pulling control-plane images from an alternative repository instead of the default `k8s.gcr.io`.
```

Comment:
```suggestion
-   New `cluster.yaml` option `spec.kubernetes.imageRepository` allows pulling control-plane images from an alternative repository instead of the default `k8s.gcr.io`.
```